rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -575,10 +638,18 @@ CFontTexture::CFontTexture(const std::string& fontfile, int size, int _outlinesi
 
 	static constexpr int FT_INTERNAL_DPI = 64;
```

Comment:
perhaps the magic constant in the review comment above is actually FT_INTERNAL_DPI?

Reply:
yes it's that same one, i'll move that somewhere else so I can also use it